frontend/src/metabase/dashboard/dashboard.js

```diff
@@ -318,12 +311,11 @@ export const fetchCardData = createThunkAction(FETCH_CARD_DATA, function(card, d
 
 export const fetchCardDuration = createThunkAction(FETCH_CARD_DURATION, function(card, datasetQuery) {
     return async function(dispatch, getState) {
-        let result = await MetabaseApi.dataset_duration(datasetQuery);
```

Comment:
Further down in this file there's the `cardDurations` reducer which has a handler for the `FETCH_CARD_DURATION` action.